diff --git a/app/build.gradle.kts b/app/build.gradle.kts
index b7121ae..e377c9b 100644
--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -227,7 +227,3 @@ println("-------------------")
 if (!gitAvailable()) {
     throw GradleException("GIT system is not available. On Windows try to run Android Studio as an Administrator. Check if GIT is installed and Studio have permissions to use it")
 }
-if (isMaster() && !allCommitted()) {
-    throw GradleException("There are uncommitted changes. Clone sources again as described in wiki and do not allow gradle update")
-}
-
diff --git a/app/src/main/AndroidManifest.xml b/app/src/main/AndroidManifest.xml
index 75ace80..3acac54 100644
--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -32,6 +32,7 @@
 
     <application
         android:name="app.aaps.MainApp"
+        android:extractNativeLibs="true"
         android:allowBackup="true"
         android:backupAgent="app.aaps.utils.SPBackupAgent"
         android:fullBackupOnly="false"
